src/main/java/uk/gov/hmcts/reform/hmc/helper/hmi/HmiSubmitHearingRequestMapper.java

```diff
@@ -21,6 +25,12 @@ public HmiSubmitHearingRequestMapper(HmiCaseDetailsMapper hmiCaseDetailsMapper,
         this.listingMapper = listingMapper;
     }
 
+    /**
+     * map hearing Request to HMI Submit Hearing Request.
+     * @param hearingId haring id
+     * @param hearingRequest hearing request
+     * @return hmiSubmitHearingRequest HMI Submit Hearing request object
+     */
```

Comment:
This (and the other JavaDoc) doesn't really add any benefit; I would just delete it as the method signature is already clear enough.

Reply:
Removing now